src/main/scala/mesosphere/marathon/core/task/tracker/impl/EmitUpdateStepImpl.scala

```diff
@@ -0,0 +1,28 @@
+package mesosphere.marathon.core.task.tracker.impl
+
+import javax.inject.Inject
+
+import mesosphere.marathon.Protos.MarathonTask
+import mesosphere.marathon.core.task.bus.{ MarathonTaskStatus, TaskStatusEmitter }
+import mesosphere.marathon.core.task.bus.TaskStatusObservables.TaskStatusUpdate
+import mesosphere.marathon.core.task.tracker.TaskStatusUpdateStep
+import mesosphere.marathon.state.{ AppDefinition, PathId, Timestamp }
+import org.apache.mesos.Protos.TaskStatus
+
+import scala.concurrent.Future
+
+/**
+  * Forward the update to the taskStatusEmitter.
+  */
+class EmitUpdateStepImpl @Inject() (taskStatusEmitter: TaskStatusEmitter) extends TaskStatusUpdateStep {
```

Comment:
In the chain of task status updates, the name `EmitUpdate` really is not very informative.